src/app/checkbox-test/checkbox-test.component.ts

```diff
@@ -0,0 +1,23 @@
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'nb-app-checkbox-test',
+  template: `
+    <nb-layout>
+      <nb-layout-column>
+        <nb-checkbox>Regular</nb-checkbox>
+        <br>
```

Comment:
let's get rid of br and replace them with `div` wrappers

Reply:
Done